CMSgov/onemac
# OY2-6411: State users and state admins can see their state accesses on profile page

Story: https://qmacbis.atlassian.net/browse/OY2-6411
Endpoint: https://d1ydton45xz752.cloudfront.net/

### Additional Notes:

- Now seeding user profile data from same JSON file locally and in the build. Let me know if that configuration needs additional tweaking to avoid hitting production with it (I am pretty sure it won't since there are existing users there, but just double and triple check the logic for me).
- Also pulling the list of users to create in Cognito from that file rather than hard coding in `deploy.sh`.

### To Test:

1. Log in as `stateuseractive@cms.hhs.local` and go to the user profile page. Confirm that both MI and VA access are active and that the cards look the way they should. Repeat with `stateuser{pending,denied,revoked}` to see their corresponding access values displayed.
2. Log in as `stateadminactiveMI@cms.hhs.local` and repeat step 1, this time confirming that the active CMS role approver is now listed in the card under the granted status. Again, repeat with the other admin test users to confirm that their accesses are set correctly.
